src/foam/nanos/u2/navigation/SignUp.js

```diff
@@ -197,13 +197,15 @@ foam.CLASS({
           this.window.history.replaceState(null, null, this.window.location.origin);
           location.reload();
         } else {
+          var user = this.subject.user;
+          this.subject = await this.auth.getCurrentSubject(null);
```

Comment:
shouldn't this go before:
line 200: var user = this.subject.user;

Reply:
how come？

Reply:
login function in signup sets the subject to the signed up user， here i am saving the signup user to be used later and resetting the subject to avoid confusion since user is not actually logged in

Reply:
@noodlemoodle add the above reply as a comment in the code.

Reply:
done @jlhughes